src/components/organisms/AdminVenueView/components/Container/Container.tsx

```diff
@@ -0,0 +1,317 @@
+      coordinatesBoundary.width,
+      onMove,
+    ]
+  );
+
+  const resizeBox = useCallback(
+    (id: string) => (dimensions: Dimensions) => {
+      if (!imageDims) return;
+
+      const { width, height } = dimensions;
+
+      onResize?.({
+        width: resizable
+          ? (coordinatesBoundary.width * width) / imageDims.width
+          : width,
+        height: resizable
+          ? (coordinatesBoundary.height * height) / imageDims.height
+          : height,
+      });
+      setBoxes(
+        update(boxes, {
+          [id]: {
+            $merge: { width, height },
+          },
+        })
+      );
+    },
+    [
+      boxes,
+      coordinatesBoundary.height,
+      coordinatesBoundary.width,
+      imageDims,
+      onResize,
+      resizable,
+    ]
+  );
+
+  const [, drop] = useDrop({
+    accept: ItemTypes.SUBVENUE_ICON,
+    drop(item: DragItem, monitor) {
+      if (!interactive) return;
+      const delta = monitor.getDifferenceFromInitialOffset() as {
+        x: number;
+        y: number;
+      };
+
+      let left = Math.round(item.left + delta.x);
+      let top = Math.round(item.top + delta.y);
+      if (snapToGrid) {
+        [left, top] = doSnapToGrid(left, top);
+      }
+
+      moveBox(item.id, left, top);
+    },
+  });
+
+  const renderSelectedRoom = (index: number) => {
+    return (
+      <>
+        <DraggableSubvenue
+          isResizable={resizable}
+          id={index.toString()}
+          imageStyle={iconImageStyle}
+          rounded={!!rounded}
+          {...boxes[index]}
+          onChangeSize={resizeBox(index.toString())}
+          lockAspectRatio={lockAspectRatio}
+        />
+        {imageDims && interactive && (
+          <CustomDragLayer
+            snapToGrid={!!snapToGrid}
+            rounded={!!rounded}
+            iconSize={boxes[Object.keys(boxes)[index]]}
+          />
+        )}
+      </>
+    );
+  };
+
+  return (
+    <div ref={drop} style={{ ...styles, ...containerStyle }}>
+      <ReactResizeDetector
+        handleWidth
+        handleHeight
+        onResize={(width, height) => setImageDims({ width, height })}
+      />
+      <img
+        alt="draggable background "
+        style={{
+          width: "100%",
+          ...backgroundImageStyle,
+        }}
+        src={backgroundImage}
+      />
+
+      {backgroundImage &&
+        rooms.map((room, index) =>
+          room === selectedRoom ? (
+            <div key={`${room.title}-${index}`}>
+              {renderSelectedRoom(index)}
+            </div>
+          ) : (
+            <div
+              className="map-preview__room"
+              key={`${room.title}-${index}`}
+              onClick={() => !selectedRoom && setSelectedRoom(room)}
+              style={{
+                position: "absolute",
+                top: `${room.y_percent}%`,
+                left: `${room.x_percent}%`,
+                width: `${room.width_percent}%`,
+                height: `${room.height_percent}%`,
```

Comment:
if you need to pass data to css, please use `useCss` as in `Section.tsx` for example

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1583/commits/f8fa0c67c34ac626919737b4bbd112b4bd715553